networktable.c

```diff
@@ -39,23 +38,40 @@ bool create_network_table(void)
 	return true;
 }
 
-// Read kernel's ARP cache using procfs
-void parse_arp_cache(void)
+bool create_network_addresses_table(void)
+{
+	bool ret;
+	// Create network_addresses table in the database
+	ret = dbquery("CREATE TABLE network_addresses ( id INTEGER NOT NULL, " \
```

Comment:
As this ID is a foreign key and not owned by the table, it should be renamed to `network_id` and have the additional property of `REFERENCING network (id)`.

If we don't have the foreign key pragma turned on, we should do that too.